OrbitGl/CaptureWindow.cpp

```diff
@@ -776,7 +776,7 @@ void CaptureWindow::RenderTimeBar() {
       std::string text = GetPrettyTime(absl::Microseconds(current_micros));
       float worldX = time_graph_.GetWorldFromUs(current_micros);
       m_TextRenderer.AddText(text.c_str(), worldX + xMargin, worldY, GlCanvas::kZValueTextUi,
-                             Color(255, 255, 255, 255));
+                             Color(255, 255, 255, 255), time_graph_.GetOverlayFontSize());
```

Comment:
See comments for `TimeGraph::GetOverlayFontSize`. I would replace this with `GParams.font_size` directly, there's no need for the TimeGraph to pass through the global font size setting for the UI.